test/Microsoft.Extensions.FileProviders.Physical.Tests/PhysicalFileProviderTests.cs

```diff
@@ -114,6 +114,23 @@ public void GetFileInfoReturnsNotFoundFileInfoForRelativePathWithEmptySegmentsTh
             }
         }
 
+        [Fact]
+        public void CreateReadStreamSuccedsOnEmptyFile()
+        {
+            using (var root = new DisposableFileSystem())
+            {
+                using (var provider = new PhysicalFileProvider(root.RootPath))
+                {
+                    var fileName = Guid.NewGuid().ToString();
+                    var filePath = Path.Combine(root.RootPath, fileName);
+                    using (File.Create(filePath)) { }
```

Comment:
`File.WriteAllBytes(filePath, new byte[0])`? A bit cleaner than the single line dispose.